Sources/ResilientDecoding/ErrorReporting.swift

```diff
@@ -1,43 +1,114 @@
-// Created by George Leontiev on 3/28/20.
+// Created by George Leontiev on 3/25/20.
```

Comment:
No good place for this, so adding here. Let's fix this typo.

<img width="974" alt="Screen Shot 2020-04-22 at 6 31 36 PM" src="https://user-images.githubusercontent.com/1791049/80049389-9916fb80-84c7-11ea-9538-8f4f6181bbb4.png">